### ISSUE TITLE
Unable to adjust image colormap

### ISSUE THREAD

**Issue by rand5** [JuliaPlots/Plots.jl#959]
When plotting a variable of type `Array{ColorTypes.Gray{Float64},2}`, the `:inferno` colormap is being used as the default with no apparent way to change to other colormaps/gradients. I have only tested this on the `pyplot()` backend.

MWE:
```
using Images, Plots
a = rand(100,100)
a = Gray.(a)
plot(a)
```

the following has no impact on the color:
```
plot(a,c=cgrad(:grays))
plot(a,palette=cgrad(:grays))
plot(a,color=cgrad(:grays))
plot(a,c=:grays)
plot(a,palette=:grays)
plot(a,color=:grays)

**Comment by rand5** [comment]
A workaround is to reset the default value for `cgrad` via `PlotUtils.default_cgrad(default = :sequential, sequential = :grays)` but this is unwieldy

**Comment by mkborregaard** [comment]
Thanks, this info should be really useful when debugging.

**Comment by mkborregaard** [comment]
(btw I think `PlotUtils.default_cgrad(default = :sequential, sequential = :grays)` could be just `default_cgrad(sequential = :grays)`)?

**Comment by appleparan** [comment]
Here is my workaround. However, It seems `Plots` can't plot `colorbar` about image
```
using ColorSchemes, Plots, Images

plot(ColorSchemes.get(ColorSchemes.inferno, N0f8.(img))
```

**Comment by mkborregaard** [comment]
why would you put a colorbar next to an image?

**Comment by appleparan** [comment]
@mkborregaard I just wanted to do same thing as [TF tutorial](https://www.tensorflow.org/tutorials/keras/classification)

**Comment by mkborregaard** [comment]
Then use `heatmap`, that is the function called `imshow` in pyplot
